<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxSerial">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>50</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Serial Connection</string>
    </property>
    <widget class="QPushButton" name="pushButtonConnect">
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>103</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>73</y>
       <width>191</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelBaud">
        <property name="text">
         <string>Baud Rate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxBaud"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="labelPort">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxPort"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButtonComListRef">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>103</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5"/>
   </widget>
   <widget class="QGroupBox" name="groupBoxConnection">
    <property name="geometry">
     <rect>
      <x>731</x>
      <y>11</y>
      <width>371</width>
      <height>671</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="title">
     <string>Control Pannel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="labelStatus">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonStatus">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>GetStatus</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBoxHorizontal">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>370</y>
       <width>131</width>
       <height>271</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Horizontal</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>91</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelHorizontalScale">
         <property name="text">
          <string>scale</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dialHorizontalScale">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>200</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>27</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="labelHorizontalShift">
         <property name="text">
          <string>position</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dialHorizontalShift_2">
         <property name="minimum">
          <number>-2500</number>
         </property>
         <property name="maximum">
          <number>2500</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBoxVertical">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>370</y>
       <width>211</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>Vertical</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>91</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelVerticalScale">
         <property name="text">
          <string>scale</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dialVerticalScale">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>13</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>9</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="labelVerticalShift">
         <property name="text">
          <string>position</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dialVeryicalShift">
         <property name="minimum">
          <number>-2500</number>
         </property>
         <property name="maximum">
          <number>2500</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>91</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="labelVerticalScale_2">
         <property name="text">
          <string>scale</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dialVerticalScale_2">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>13</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>9</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="labelVerticalShift_3">
         <property name="text">
          <string>position</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dialVeryicalShift_2">
         <property name="minimum">
          <number>-2500</number>
         </property>
         <property name="maximum">
          <number>2500</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>229</x>
       <y>259</y>
       <width>131</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Trigger</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QDial" name="dialHorizontalTrigger">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>200</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>711</width>
      <height>671</height>
     </rect>
    </property>
   </widget>
   <zorder>groupBoxConnection</zorder>
   <zorder>groupBoxSerial</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>customPlot</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
